client/signup/steps/domains/style.scss

```diff
@@ -551,25 +588,48 @@ body.is-section-signup {
 		@include break-medium {
 			max-width: 290px;
 		}
+
+		@include break-large {
+			margin-left: 40px;
+		}
+
+		@include break-huge {
+			margin-left: 60px;
+		}
+
+		&.fade-out {
+			display: none;
+		}
 	}
 
-	.domains__domain-side-content.fade-out {
-		display: none;
+	.step-container-v2__heading {
+		h1:not(.small) {
+			letter-spacing: -0.4px;
+			/* stylelint-disable-next-line declaration-property-unit-allowed-list */
+			line-height: 1.2em;
```

Comment:
https://github.com/Automattic/wp-calypso/blob/7800534c183e0ccde34b18b922ee0d825177cceb/client/components/formatted-header/style.scss#L110-L111

Since we're replicating this referenced style in here, we need to disable the rule as well too.